RoboJackets/robocup-software
# Situation Analysis actual enable

## Description
This pull request does a variety of things to get the minimum viable product of situation analysis.

1. Moves all plays into directories based on their primary situation
2. Modified situation analysis and a number of plays for scoring and generally smoothing out functionality
3. Created play variants to cover situations, including adaptive formation.

## Steps to test
### Test Case 1
1. Load the Situation Analysis playbook
2. Observe game play

Note that playbook changes and situation changes will be needed before 

Expected result: We switch plays appropriately and it's relatively smooth and good

## General

This is a pretty big change and I would like a lot of people to look at it. I'm kind of just throwing this out there to get feedback. Please take a look at it and run it, its really exciting to watch the play switching.

I also can't make pretty on my new laptop, so that is a thing. I expect there to be changes.
